### Build Failed!

The last (useful) 30 log lines are shown below:
[View Full Log](https://console.aws.amazon.com/cloudwatch/home?region=ap-southeast-2#logEvent:group=/aws/codebuild/assignar-ci;stream=19a661f9-c30a-419d-abf3-a48a99d39e31)
```
$ make ci
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-base/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-graphql/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-node/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-react/tsconfig.json
$ eslint --cache ./packages --ext .ts

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/eslint-base.ts
  8:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/eslint-comments.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/import.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/jest.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/typescript.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-react/src/plugins/import.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-react/src/plugins/react.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

✖ 7 problems (7 errors, 0 warnings)
  7 errors and 0 warnings potentially fixable with the `--fix` option.
```
<details><summary>Unfiltered log</summary>
<p>

```
[Container] 2018/09/11 00:05:14 Waiting for agent ping
[Container] 2018/09/11 00:05:18 Waiting for DOWNLOAD_SOURCE
[Container] 2018/09/11 00:05:21 Phase is DOWNLOAD_SOURCE
[Container] 2018/09/11 00:05:21 CODEBUILD_SRC_DIR=/codebuild/output/src922956593/src/github.com/assignar/ci
[Container] 2018/09/11 00:05:21 YAML location is /codebuild/output/src922956593/src/github.com/assignar/ci/buildspec.yml
[Container] 2018/09/11 00:05:21 Processing environment variables
[Container] 2018/09/11 00:05:21 Decrypting parameter store environment variables
[Container] 2018/09/11 00:05:21 Moving to directory /codebuild/output/src922956593/src/github.com/assignar/ci
[Container] 2018/09/11 00:05:21 Registering with agent
[Container] 2018/09/11 00:05:21 Phases found in YAML: 1
[Container] 2018/09/11 00:05:21  BUILD: 1 commands
[Container] 2018/09/11 00:05:21 Phase complete: DOWNLOAD_SOURCE Success: true
[Container] 2018/09/11 00:05:21 Phase context status code:  Message:
[Container] 2018/09/11 00:05:21 Entering phase INSTALL
[Container] 2018/09/11 00:05:21 Phase complete: INSTALL Success: true
[Container] 2018/09/11 00:05:21 Phase context status code:  Message:
[Container] 2018/09/11 00:05:22 Entering phase PRE_BUILD
[Container] 2018/09/11 00:05:22 Phase complete: PRE_BUILD Success: true
[Container] 2018/09/11 00:05:22 Phase context status code:  Message:
[Container] 2018/09/11 00:05:22 Entering phase BUILD
[Container] 2018/09/11 00:05:22 Running command node ./test.js
Cloning into 'eslint-config-assignar'...
Note: checking out 'a557194c14da84def057d91afa87629530f7be34'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a557194... [ENG] update rules to match monorepo


node version: v8.10.0
yarn version: 1.5.1


yarn install v1.5.1
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
Done in 8.62s.


yarn run v1.5.1
$ make ci
make[1]: Entering directory '/tmp/eslint-config-assignar'
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-base/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-graphql/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-node/tsconfig.json
$ /tmp/eslint-config-assignar/node_modules/.bin/tsc -p ./packages/eslint-config-assignar-react/tsconfig.json
make[1]: Leaving directory '/tmp/eslint-config-assignar'
make[1]: Entering directory '/tmp/eslint-config-assignar'
$ eslint --cache ./packages --ext .ts

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/eslint-base.ts
  8:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/eslint-comments.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/import.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/jest.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-base/src/plugins/typescript.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-react/src/plugins/import.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

/tmp/eslint-config-assignar/packages/eslint-config-assignar-react/src/plugins/react.ts
  1:13  error  Unexpected space before the ':'  typescript/type-annotation-spacing

✖ 7 problems (7 errors, 0 warnings)
  7 errors and 0 warnings potentially fixable with the `--fix` option.

error An unexpected error occurred: "Command failed.
Exit code: 1
Command: sh
Arguments: -c eslint --cache ./packages --ext .ts
Directory: /tmp/eslint-config-assignar
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "/tmp/eslint-config-assignar/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Makefile:9: recipe for target 'lint' failed
make[1]: *** [lint] Error 1
make[1]: Leaving directory '/tmp/eslint-config-assignar'
Makefile:30: recipe for target 'ci' failed
make: *** [ci] Error 2
error An unexpected error occurred: "Command failed.
Exit code: 2
Command: sh
Arguments: -c make ci
Directory: /tmp/eslint-config-assignar
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "/tmp/eslint-config-assignar/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
child_process.js:644
    throw err;
    ^

Error: Command failed: yarn ci
    at checkExecSyncError (child_process.js:601:13)
    at Object.execSync (child_process.js:641:13)
    at execSync (/codebuild/output/src922956593/src/github.com/assignar/ci/test.js:7:18)
    at Object.<anonymous> (/codebuild/output/src922956593/src/github.com/assignar/ci/test.js:108:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)

[Container] 2018/09/11 00:05:47 Command did not exit successfully node ./test.js exit status 1
[Container] 2018/09/11 00:05:47 Phase complete: BUILD Success: false
[Container] 2018/09/11 00:05:47 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: node ./test.js. Reason: exit status 1
[Container] 2018/09/11 00:05:47 Entering phase POST_BUILD
[Container] 2018/09/11 00:05:47 Phase complete: POST_BUILD Success: true
[Container] 2018/09/11 00:05:47 Phase context status code:  Message:
```

</p>
</details>

[Click here to retry the build](https://ut2fgw3cae.execute-api.ap-southeast-2.amazonaws.com/development//retry-build?repo=eslint-config-assignar&commit=a557194c14da84def057d91afa87629530f7be34&retryToken=4dce19bb-302e-4999-9a64-e651e47afc9a)

--- a/packages/eslint-config-assignar-base/src/plugins/eslint-base.ts
+++ b/packages/eslint-config-assignar-base/src/plugins/eslint-base.ts
@@ -5,7 +5,7 @@ import globals = require('eslint-restricted-globals')
 const pseudoAllowedGlobals = ['name', 'status', 'location', 'open', 'close', 'event']
 const restrictedGlobals = globals.filter(g => pseudoAllowedGlobals.indexOf(g) === -1)
 
-const rules : Rules.Eslint = {
+const rules: Rules.Eslint = {
     // This rule warns if setters are defined without getters
     // https://eslint.org/docs/rules/accessor-pairs
     'accessor-pairs': [

--- a/packages/eslint-config-assignar-base/src/plugins/eslint-comments.ts
+++ b/packages/eslint-config-assignar-base/src/plugins/eslint-comments.ts
@@ -1,4 +1,4 @@
-const rules : Rules.EslintComments = {
+const rules: Rules.EslintComments = {
     // require a eslint-enable comment for every eslint-disable comment
     // https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/disable-enable-pair.html
     'eslint-comments/disable-enable-pair': [

--- a/packages/eslint-config-assignar-base/src/plugins/import.ts
+++ b/packages/eslint-config-assignar-base/src/plugins/import.ts
@@ -1,4 +1,4 @@
-const rules : Rules.Import = {
+const rules: Rules.Import = {
     // ensure default import coupled with default export
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/default.md
     'import/default': 'error',
@@ -175,7 +175,7 @@ const rules : Rules.Import = {
 
     // Ensures that there are no useless path segments
     // https://github.com/benmosher/eslint-plugin-import/issues/1032
-    'import/no-useless-path-segments': 'error',
+    'import/no-useless-path-segments': 'off',
 
     // Forbid Webpack loader syntax in imports
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-webpack-loader-syntax.md

--- a/packages/eslint-config-assignar-base/src/plugins/jest.ts
+++ b/packages/eslint-config-assignar-base/src/plugins/jest.ts
@@ -1,4 +1,4 @@
-const rules : Rules.Jest = {
+const rules: Rules.Jest = {
     // This rule gives you control over the usage of test/it keywords in your codebase
     // https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md
     'jest/consistent-test-it': [

--- a/packages/eslint-config-assignar-base/src/plugins/typescript.ts
+++ b/packages/eslint-config-assignar-base/src/plugins/typescript.ts
@@ -1,4 +1,4 @@
-const rules : Rules.Typescript = {
+const rules: Rules.Typescript = {
     // Grouping overloaded members together can improve readability of the code.
     // https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/adjacent-overload-signatures.md
     'typescript/adjacent-overload-signatures': 'error',

--- a/packages/eslint-config-assignar-react/src/plugins/import.ts
+++ b/packages/eslint-config-assignar-react/src/plugins/import.ts
@@ -1,4 +1,4 @@
-const rules : Partial<Rules.Import> = {
+const rules: Partial<Rules.Import> = {
     // No Node.js builtin modules
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-nodejs-modules.md
     'import/no-nodejs-modules': 'error',

--- a/packages/eslint-config-assignar-react/src/plugins/react.ts
+++ b/packages/eslint-config-assignar-react/src/plugins/react.ts
@@ -1,4 +1,4 @@
-const rules : Rules.React = {
+const rules: Rules.React = {
     // Enforces consistent naming for boolean props
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming
     'react/boolean-prop-naming': 'off',

